docs/bc-registries-audit-notification-workflow.md

```diff
@@ -88,4 +88,17 @@ Specifics regarding the commands referenced in this workflow can be found here:
 
 ## Audit scripts for Aries VCR/OrgBook and BC Registries Issuer
 
-This repository provides scripts to audit the OrgBook search database and agent wallet against the source BC Registries data; [Audit Scripts](https://github.com/bcgov/von-bc-registries-audit/blob/main/README.md#understanding-the-output)
\ No newline at end of file
+This repository provides scripts to audit the OrgBook search database and agent wallet against the source BC Registries data; [Audit Scripts](https://github.com/bcgov/von-bc-registries-audit/blob/main/README.md#understanding-the-output)
+
+## Troubleshoot Failed Credentials Error
+
+If there are companies that dont't get re-processed because the credentials were created in the credential_log table in the event processor database. Follow the steps below to troubleshoot failed credentials error:
+
+Run the `./manage -e prod runPipeline` command and verify the credentials errors. An example regarding this is shown below in the image:
```

Comment:
```suggestion
Run the `./manage -e prod getPipelineStatus` command and verify the credentials errors. An example regarding this is shown below in the image:
```

Sorry I missed this the first time.

For consistency and ease of reuse, please update `bc-registries-agent-configurations-manage-script.md`.  Add a section for `./manage -e prod getPipelineStatus` describing it's use and link to it from here.  i.e. Follow the same pattern used with the `requeueOrganization`, `queueOrganization`, and `requeueFailedCreds` commands.

The `./manage -e prod runPipeline` command is what you run at the end after you have requeued any records using the other commands.  You should add this to the documentation and make a section for it in the `bc-registries-agent-configurations-manage-script.md` as well so you can link to it.